src/foam/nanos/controller/ApplicationController.js

```diff
@@ -499,19 +510,25 @@ foam.CLASS({
       });
     },
 
-    function render() {
-      var self = this;
-      self.addMacroLayout();
+    async function render() {
```

Comment:
Dont think this change to render is required or current, is this a merge carryover?